### DISCUSSION TITLE
Kraje

### DISCUSSION THREAD

**Discussion by Rogall4** [TomaszGasior/RadioLista-v3#165] [2. Pomysły Ideas]
Witam. Ja dodam od siebie niby taki niuans ale często przy wpisywaniu brakuje mi do wyboru większej ilości krajów. Ograniczone jest do nazwy sąsiadów a możliwości nasłuchu często przy tropo wykraczają na większy obszar Europy. Jeżeli jest możliwość to proszę do zakładki "Kraj" dodać: Dania, Szwecja, Węgry, Austria. Pozdrawiam.

**Comment by TomaszGasior** [comment]
Dziękuję ci za twoją propozycję — masz rację. Lista podpowiedzi została stworzona przeze mnie wiele lat temu i jest bardzo niedoskonałym rozwiązaniem. Jednakże, zamiast dodawać do niej nowej pozycje, problem chciałbym rozwiązać w zupełnie inny sposób opisany w zadaniu https://github.com/TomaszGasior/RadioLista-v3/issues/97. Mianowicie docelowo RadioLista zostanie pozbawiona jakichkolwiek wpisanych „na sztywno” podpowiedzi, a zamiast tego będą one generowane dynamicznie na podstawie wartości już wpisanych w danej kolumnie/polu w innych stacjach tego samego wykazu.

Możesz zasubskrybować zalinkowane zadanie https://github.com/TomaszGasior/RadioLista-v3/issues/97, aby być na bieżąco z postępem. Postaram się zaimplementować rozwiązanie w tym roku.

**Comment by UniqueTM2** [comment]
Co do podpowiedzi, hmmm, różnie kazdy uzywa, część osób korzysta z pełnej nazwy kraju, ja korzystam z oznaczeń skrótowych, bo z racji rozbudowanej ilości kolumn brakuje mi miejsca. W sumie z tych wstepnie wpisanych nie korzystam. Ewentualnie mozna wpisac do bazy liste popularniejszych krajów ościennych mozliwych do odbioru w naszym regionie, ale w odpowiedniej formie. Ja uzywam według kodu alfa-3

**Reply by TomaszGasior** [reply]
Jednym z powodów wybrania rozwiązania, które opisałem wyżej, był właśnie fakt, że różne osoby zapisują kraje, nazwy stacji i inne tego typu pola we własny, specyficzny sposób. Tworzenie predefiniowanej listy podpowiedzi nie ma więc jakiegokolwiek sensu.

**Comment by TomaszGasior** [comment]
Problem zostanie rozwiązany w wersji v3.39 RadioListy, która wskoczy jutro wieczorem na serwer. https://github.com/TomaszGasior/RadioLista-v3/discussions/173
